.claude/hooks/pr-worktree.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Shouldn't there also be a `settings.json` that uses this hook? I don't see it reference anywhere in the git hooks.